### DISCUSSION TITLE
How to add linting to your Teams Bot?

### DISCUSSION THREAD

**Discussion by IAM5K** [OfficeDev/microsoft-365-agents-toolkit#11674] [Q&A]
We are creating a bot and based on our company coding standard it is mandatory that we implement coding standards or linting in our project what is the standard plugin package or linter that we use for linting bot framework applications.

I have already tried using eslint and prettier but they have some conflicts and package issues.
I found using mslint in a repository but lost it. Is there some guideline that we use to add coding standards and checks (linting) to our bot project.

**Comment by MuyangAmigo** [comment]
Hey @IAM5K, I'm transferring this to an issue and will loop other teams to assist you.

**Comment by MuyangAmigo** [comment]
Transferred to issue: https://github.com/OfficeDev/teams-toolkit/issues/11727

**Reply by IAM5K** [reply]
Thanks
